h/static/scripts/services.coffee

```diff
@@ -69,13 +61,13 @@ class Hypothesis extends Annotator
     whitelist = ['target', 'document', 'uri']
     this.addPlugin 'Bridge',
       gateway: true
-      formatter: (annotation) =>
+      formatter: (annotation) ->
         formatted = {}
         for k, v of annotation when k in whitelist
           formatted[k] = v
         formatted
-      parser: (annotation) =>
-        parsed = {}
+      parser: (annotation) ->
+        parsed = new store.annotation()
```

Comment:
I really think we need to have this renamed to `store.AnnotationResource()` and `store.SearchResource`.